src/lib.rs

```diff
@@ -22,7 +22,7 @@
 //!
 
 #![warn(missing_docs)]
-
+#![allow(clippy::trivially_copy_pass_by_ref)]
```

Comment:
What do we need this for?

Reply:
We discussed this in discord, Clippy showed '& self' in a function as an error and suggested me to change it too 'self' which I don't want because it would consume self. I added this lint to prevent the CI from braking.